Example/IGListKitExamples/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -2,6 +2,16 @@
   "images" : [
     {
       "idiom" : "iphone",
+      "size" : "20x20",
```

Comment:
Had told it not to commit this 🤔
Xcode be really wanting this change to happen!